content/committees/specification/tckprocess/index.md

```diff
@@ -46,16 +46,24 @@ namespace format of `ee.jakarta.tck.<spec-name>` is recommended.
 
 ## TCK types and requirements
 
-There are 3 different types of TCKs, detailed below.
-- Type 1: Tests that only apply when running outside of a platform (or profile), such as on Java SE, to test behavior defined by the specification for that mode of execution.
-- Type 2: Tests that only apply when running in a platform (or profile) to test behavior defined by the specification for that mode of execution.
-- Type 3: Tests that apply to the behavior defined by a specification for the execution either as a standalone or in a platform (or profile).
+TCKs for individual specifications can include tests of three different types of specification requirements, detailed below:
 
-The individual specification TCK must contain Type 2 or Type 3 tests while Type 1 tests are optional. A profile-ready mechanism must be provided for running the required TCK tests (types 2 and 3) in the profiles they are included in: platform, web profile or core profile. Type 1 tests must be excluded and not required to be executed.
+- Type 1: Behavior defined by the specification that is _only_ applicable when the implementation is running outside of a platform (or profile), such as on Java SE, to test behavior defined by the specification for that mode of execution.
+- Type 2: Behavior defined by the specification that is _only_ applicable when the implementation is running in a platform (or profile).
+- Type 3: Behavior defined by the specification that is applicable _both_ when the implementation is running outside of a platform (or profile) _and_ when it is running in a platform (or profile).
 
-The individual specification TCK must provide a mechanism for running the required TCK tests (types 1 and 3) outside the context of a platform or profile. Type 2 tests must be excluded and not run.
+Not all individual specifications will have Type 1 or Type 2 requirements. The individual specification TCK should include tests of all types of requirements and must contain tests of Type 2 requirements, if they exist, and of Type 3 requirements.
```

Comment:
This line is meant to clarify L54 "The individual specification TCK must contain Type 2 or Type 3 tests while Type 1 tests are optional."

I went with this because the existing text could be interpreted to mean if Type 1 requirements exist, tests of them are optional. If that was the intent I didn't want to change it. But if that wasn't the intent I'd prefer to say:

"Not all individual specifications will have Type 1 or Type 2 requirements. For all Types of requirements found in a specification, the individual specification TCK must include tests."

(I use "Types" with a capital T there to try and avoid implying that all requirements found in the specification must be covered in the TCK.)

Also, FWIW *in theory* a spec might only have Type 1 and Type 2 requirements but no Type 3. But I ignore that because it's not a realistic scenario, and discussing it would just add confusion.